openrtb_ext/imp_freewheelssp.go

```diff
@@ -1,5 +1,7 @@
 package openrtb_ext
 
+import "encoding/json"
+
 type ImpExtFreewheelSSP struct {
-	ZoneId int `json:"zoneId"`
+	ZoneId json.Number `json:"zoneId"`
```

Comment:
you could use `jsonutil.StringInt` for [this change. ](https://github.com/prebid/prebid-server/blob/a1f48b54af8e831927a0f6838f56493fa03efb08/util/jsonutil/stringInt.go#L7-L8)